eng/tools/OneLocBuildSetup/OneLocBuildSetup.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
-    <IncludeVSSDKPackages>false</IncludeVSSDKPackages>
```

Comment:
This is the only place we set this property.

Reply:
Yeah, these packages were causing issues with building that tool.